### ISSUE TITLE
Notification icon/badge

### ISSUE THREAD

**Issue by nicolas2bert** [scality/core-ui#13]
<img width="108" alt="Screen Shot 2019-04-18 at 2 35 04 PM" src="https://user-images.githubusercontent.com/7453636/56392799-4be2b880-61e7-11e9-8e7c-1cc1168b1164.png">

**Comment by felipebrunetti94** [comment]
Hi @nicolas2bert , what is the problem, how can I contribute on this issue ? 
It is just to create this component ?

**Comment by nicolas2bert** [comment]
It seems components need to be created. Please, check with @carlito-scality.

**Comment by carlito-scality** [comment]
> It seems components need to be created. Please, check with @carlito-scality.

Yes, I think with the new design we are working in progress. This component should look like:
![image](https://user-images.githubusercontent.com/47394132/70601771-1a45b400-1bf3-11ea-83e5-c4bbb394cb6d.png)
![image](https://user-images.githubusercontent.com/47394132/70602199-22522380-1bf4-11ea-859f-80dcccc2897a.png)


@felipebrunetti94 please wait until the new design is validated ;)
